client/packages/invoices/src/Prescriptions/DetailView/SidePanel/SidePanel.tsx

```diff
@@ -1,34 +1,64 @@
 import React, { memo } from 'react';
 import {
   CopyIcon,
+  DeleteIcon,
   DetailPanelAction,
   DetailPanelPortal,
   useNotification,
+  useSingleDeleteConfirmation,
   useTranslation,
 } from '@openmsupply-client/common';
 import { usePrescription } from '../../api';
 import { AdditionalInfoSection } from './AdditionalInfoSection';
 import { PricingSection } from './PricingSection';
+import { canDeleteInvoice } from 'packages/invoices/src/utils';
```

Comment:
the package import breaks tests. and we are already in 'invoices'!
```suggestion
import { canDeleteInvoice } from '../../../utils';
```

side note: should really be called `canDeleteOutboundShipment` since the fn is type specific 🤷